game.js

```diff
@@ -9,6 +9,7 @@ export const game = {
   gamesCounter: 1,
   winnings: {},
   winningStates: [],
+  alerts: [["The board still empty!"], ["No winnings yet"], ["No saved game"]]
```

Comment:
i like the idea of saving alerts in constant global scope... 
but why this nested array? you can simply do : ['msg1','msg2'] no need this [['msg1'],['msg2']] approach.
better idea will be to put the in object:
alert:{
emptyBoard:'the board still empty',
noWinningYet:'No winning yet'... etc
}
this way you don't need to hardcoded the index of the array ...

Reply:
right! i'll go with the alerts object